### PR TITLE
jay-edit

### PR DESCRIPTION
**Pull request by JaydeepRokade**
spellings - grammer

### CODE REVIEW

**Comment by JaydeepRokade** [comment]
Done with changes

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1>Jobs</h1>
     <div class="row">
         <div class="col-md-5">
           <p>
-            Need a logo designed, a usability study, or an interface-facelift?
+            Required a logo designed, a usability study, or an interface-facelift?
```
Comment (Erioldoesdesign) [inline index.html:15]: This language isn't better than the original, it's a little confusing!

File: job-form.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 	<h1>Submit a Job</h1>
 	<p class="lead">If you work on a <a href="#floss" title="Free as in Speech (and maybe Beer)">Free/Libre and Open Source<sup>*</sup></a> project and you need
 some design help, let the Open Source Design community know that you are looking
-for design contributions or to hire a designer. Your posting will be displayed on our <a
+for design contributions or to hire a designer. Your posts will be displayed on our Web pages and magzine<a
```
Comment (Erioldoesdesign) [inline job-form.html:13]: Uhhh we don't have a magazine webpage so this is strange addition

**Comment by Erioldoesdesign** [comment]
@JaydeepRokade I looked over your wording suggestions and to me, they don't improve the wording and add some inaccurate links and references to pages that we do not have in our website - I'll be not merging this PR and closing it.

**Pull request closed without merging**